pull-requests.md

```diff
@@ -0,0 +1,206 @@
+# Pull Request submission and review guidelines
+
+## Purpose of this document
+
+The objective of this document is to have a single place with information from the following pages:
+
+- **PR etiquette:** https://github.com/mdn/content#pull-request-etiquette
+- **Reviewing PRs:** https://github.com/mdn/content/blob/main/REVIEWING.md
+- **Open source etiquette:** https://developer.mozilla.org/en-US/docs/MDN/Contribute/Open_source_etiquette
+
+## Contributing to MDN
+
+This section describes how contributors make changes on MDN Web Docs and how the changes are reviewed and land on the site.
+Content changes to MDN Web Docs are related to a variety of work streams, including:
+
+- **Day-to-day content improvements:** documentation of new APIs, CSS properties, platform updates and content additions.
+  This is usually done by MDN Web Docs staff working for Mozilla, Google, Open Web Docs, Samsung, but also by community volunteers.
+- **Minor fixes:** small updates done to the site to fix typos, grammatical issues, and technical inaccuracies.
+  These issues are usually found by users of MDN Web Docs.
+- **Content bug fixes:** usually done by volunteers to close issues on `mdn/content` repo.
+
+Regardless of how content changes are done, they are submitted as pull requests (PRs) on GitHub:
+
+1. **Reviewers** comprised of MDN staff members and volunteers are "topic review owners".
+   When a pull request related to a particular topic is opened, it will be assigned to that area's topic review owners.
+   The `CODEOWNERS` file specifies the teams who own certain content directories.
+2. **Approved pull requests** can be merged after a approving review has been made.
+   The assigned reviewer will merge the pull request.
+3. **Content update** is done automatically via a site rebuild once every 24 hours.
+   Volunteers can see their changes go live within a day of merging.
```

Comment:
What would be a good name for this stage that aligns with 'Submission' > 'Review' > 'Approval'?
Or perhaps we don't call this a stage..

```suggestion
Updated content goes live within a day of merging via a site rebuild once every 24 hours.
```

Reply:
"Publishing" maybe